bittensor/utils/wallet_utils.py

```diff
@@ -8,7 +8,7 @@
 # the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
 # and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
-# The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+# The above copyright notice and this permission notice shall be included in all copies or large portions of
```

Comment:
You can't change wording on the license.

Reply:
Done